modules/analysis/src/service/load.rs

```diff
@@ -323,19 +327,25 @@ impl AnalysisService {
         // Set the result. A parallel call might have done the same. We wasted some time, but the
         // state is still correct.
 
-        self.graph.write().insert(distinct_sbom_id.to_string(), g);
+        let g = Arc::new(g);
+        self.graph_cache
```

Comment:
it occurred to me there might be times when we want to suspend caching (might be useful for debugging for example)... not sure if this is important or not to consider for right now